src/main/java/io/dockstore/provision/ICGCStorageClientPlugin.java

```diff
@@ -72,40 +69,64 @@ public void setConfiguration(Map<String, String> map) {
         }
 
         public boolean downloadFrom(String sourcePath, Path destination) {
-            String client = "/icgc/dcc-storage/bin/dcc-storage-client";
-            if (config.containsKey(DCC_CLIENT_KEY)) {
-                client = config.get(DCC_CLIENT_KEY);
-            }
+            String clientKey = getKey();
 
             // ambiguous how to reference synapse files, rip off these kinds of headers
             String prefix = "icgc://";
             if (sourcePath.startsWith(prefix)){
                 sourcePath = sourcePath.substring(prefix.length());
+            } else {
+                System.err.println("File prefix not handled by this plugin.");
```

Comment:
In theory this should never occur with the normal file provisioning as this should only get invoked for values starting returned by `schemesHandled`. But I would call this `protocol` or `scheme` instead of `file prefix`.